packages/ledger/src/lib/ledger-client.ts

```diff
@@ -60,6 +60,7 @@ export interface Subscription {
 
 // Not using TransportWebHID.isSupported as it's chosen to use a Promise...
 export const isLedgerSupported = () => {
+  // @ts-ignore
```

Comment:
I think this is simply missing a type package around Ledger that injects `window.navigator.hid` into the TypeScript world (`@types/w3c-web-hid`). Maybe it's missing now because we've updated the package by accident? Not a major issue to explicitly import it for now, but worth investigating why this is now an issue

Reply:
Duplicate of https://github.com/near/wallet-selector/pull/354#discussion_r924602649